### DISCUSSION TITLE
Who makes different human outfits?

### DISCUSSION THREAD

**Discussion by warp-core** [endless-sky/endless-sky#7105] [Lore]
Something on Discord got me started wondering which outfitter designed/makes different outfits in human space.
There are five different outfit manufacturers in human space (that we know of):
- Deep Sky
- Delta V
- Kraz Cybernetics
- Lovelace Labs
- Syndicate(d Systems?)

I made the game produce a list of every outfit that appears in an outfitter with every outfitter it is listed in to help narrow it down, and I've assigned designers/manufacturers to most of hte outfits, but there are some that aren't entirely clear:

<details><summary>Rather large table</summary>

name | designer/manufacturer | outfitters |   |   |   |   |   |   |   |   |   |   |   |   |  
-- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | --
A120 Atomic Thruster | Deep Sky | Deep Sky Advanced | Deep Sky Basics | Lovelace Advanced |   |   |   |   |   |   |   |   |   |   |  
A125 Atomic Steering | Deep Sky | Deep Sky Advanced | Deep Sky Basics | Lovelace Advanced |   |   |   |   |   |   |   |   |   |   |  
A250 Atomic Thruster | Deep Sky | Deep Sky Advanced | Deep Sky Basics |   |   |   |   |   |   |   |   |   |   |   |  
A255 Atomic Steering | Deep Sky | Deep Sky Advanced | Deep Sky Basics |   |   |   |   |   |   |   |   |   |   |   |  
A370 Atomic Thruster | Deep Sky | Deep Sky Advanced | Deep Sky Basics |   |   |   |   |   |   |   |   |   |   |   |  
A375 Atomic Steering | Deep Sky | Deep Sky Advanced | Deep Sky Basics |   |   |   |   |   |   |   |   |   |   |   |  
A520 Atomic Thruster | Deep Sky | Deep Sky Advanced |   |   |   |   |   |   |   |   |   |   |   |   |  
A525 Atomic Steering | Deep Sky | Deep Sky Advanced |   |   |   |   |   |   |   |   |   |   |   |   |  
A860 Atomic Thruster | Deep Sky | Deep Sky Advanced |   |   |   |   |   |   |   |   |   |   |   |   |  
A865 Atomic Steering | Deep Sky | Deep Sky Advanced |   |   |   |   |   |   |   |   |   |   |   |   |  
AR120 Reverse Thruster | Deep Sky | Deep Sky Advanced | Deep Sky Basics |   |   |   |   |   |   |   |   |   |   |   |  
Anti-Missile Turret | Lovelace Labs | Lovelace Advanced | Lovelace Basics |   |   |   |   |   |   |   |   |   |   |   |  
Armageddon Core | Lovelace Labs | Lovelace Advanced |   |   |   |   |   |   |   |   |   |   |   |   |  
Asteroid Scanner |   | Common Outfits |   |   |   |   |   |   |   |   |   |   |   |   |  
Beam Laser | Deep Sky | Basic Weapons | Common Outfits | Deep Sky Advanced | Deep Sky Basics | Kraz Advanced | Kraz Basics |   |   |   |   |   |   |   |  
Blaster Turret | Syndicate | Basic Weapons | Common Outfits | Kraz Advanced | Kraz Basics | Syndicate Advanced | Syndicate Basics |   |   |   |   |   |   |   |  
Breeder Reactor | Delta V | Delta V Advanced |   |   |   |   |   |   |   |   |   |   |   |   |  
Brig |   | Delta V Basics | Kraz Basics | Pirate Outfits |   |   |   |   |   |   |   |   |   |   |  
Bullet Boxes |   | Pirate Outfits |   |   |   |   |   |   |   |   |   |   |   |   |  
Bunk Room |   | Coalition Advanced | Coalition Basics | Common Outfits | Hai Advanced | Hai Intermediate | Korath Basics | Korath Exiles | Remnant | Wanderer Advanced |   |   |   |   |  
Caldera Afterburner | Delta V | Delta V Advanced | Delta V Basics | Kraz Advanced | Pirate Outfits |   |   |   |   |   |   |   |   |   |  
Capybara Reverse Thruster | Delta V | Delta V Advanced | Delta V Basics |   |   |   |   |   |   |   |   |   |   |   |  
Cargo Expansion |   | Coalition Advanced | Coalition Basics | Common Outfits | Hai Advanced | Hai Basics | Hai Intermediate | Korath Basics | Korath Exiles | Remnant | Unfettered Basics | Wanderer Advanced | Wanderer Basics |   |  
Cargo Scanner |   | Basic Outfits | Common Outfits | Hai Advanced | Hai Intermediate | Kraz Advanced | Kraz Basics | Lovelace Basics | Remnant |   |   |   |   |   |  
Chipmunk Plasma Steering | Delta V | Delta V Advanced | Delta V Basics | Kraz Advanced | Kraz Basics | Syndicate Advanced | Syndicate Basics |   |   |   |   |   |   |   |  
Chipmunk Plasma Thruster | Delta V | Delta V Advanced | Delta V Basics | Kraz Advanced | Kraz Basics | Syndicate Advanced | Syndicate Basics |   |   |   |   |   |   |   |  
Cooling Ducts |   | Basic Outfits | Common Outfits |   |   |   |   |   |   |   |   |   |   |   |  
D14-RN Shield Generator | Deep Sky | Basic Outfits | Common Outfits |   |   |   |   |   |   |   |   |   |   |   |  
D23-QP Shield Generator | Deep Sky | Common Outfits | Deep Sky Advanced | Deep Sky Basics | Kraz Advanced | Kraz Basics | Lovelace Advanced | Lovelace Basics | Syndicate Advanced | Syndicate Basics |   |   |   |   |  
D41-HY Shield Generator | Deep Sky | Common Outfits | Deep Sky Advanced | Deep Sky Basics | Kraz Advanced | Syndicate Advanced |   |   |   |   |   |   |   |   |  
D67-TM Shield Generator | Deep Sky | Deep Sky Advanced | Syndicate Advanced |   |   |   |   |   |   |   |   |   |   |   |  
D94-YV Shield Generator | Deep Sky | Deep Sky Advanced |   |   |   |   |   |   |   |   |   |   |   |   |  
Dwarf Core | Deep Sky | Deep Sky Advanced |   |   |   |   |   |   |   |   |   |   |   |   |  
Energy Blaster | Syndicate | Basic Outfits | Basic Weapons | Common Outfits | Kraz Advanced | Kraz Basics |   |   |   |   |   |   |   |   |  
Fission Reactor | Syndicate | Syndicate Advanced |   |   |   |   |   |   |   |   |   |   |   |   |  
Fragmentation Grenades |   | Pirate Outfits |   |   |   |   |   |   |   |   |   |   |   |   |  
Fuel Pod |   | Common Outfits | Hai Advanced | Hai Basics | Hai Intermediate | Remnant | Remnant Core | Unfettered Basics | Wanderer Advanced | Wanderer Basics |   |   |   |   |  
Fusion Reactor | Lovelace Labs | Lovelace Advanced |   |   |   |   |   |   |   |   |   |   |   |   |  
Gatling Gun |   | Pirate Outfits |   |   |   |   |   |   |   |   |   |   |   |   |  
Gatling Gun Ammo |   | Pirate Outfits |   |   |   |   |   |   |   |   |   |   |   |   |  
Greyhound Plasma Steering | Delta V | Delta V Advanced | Delta V Basics | Syndicate Advanced |   |   |   |   |   |   |   |   |   |   |  
Greyhound Plasma Thruster | Delta V | Delta V Advanced | Delta V Basics | Syndicate Advanced |   |   |   |   |   |   |   |   |   |   |  
Heavy Anti-Missile Turret | Lovelace Labs | Lovelace Advanced |   |   |   |   |   |   |   |   |   |   |   |   |  
Heavy Laser | Deep Sky | Deep Sky Advanced | Deep Sky Basics | Kraz Advanced |   |   |   |   |   |   |   |   |   |   |  
Heavy Laser Turret | Deep Sky | Deep Sky Advanced |   |   |   |   |   |   |   |   |   |   |   |   |  
Heavy Rocket | Delta V | Ammo South | Delta V Advanced | Delta V Basics |   |   |   |   |   |   |   |   |   |   |  
Heavy Rocket Launcher | Delta V | Delta V Advanced |   |   |   |   |   |   |   |   |   |   |   |   |  
Heavy Rocket Pod | Delta V | Delta V Advanced |   |   |   |   |   |   |   |   |   |   |   |   |  
Heavy Rocket Rack | Delta V | Delta V Advanced |   |   |   |   |   |   |   |   |   |   |   |   |  
Hyperdrive |   | Basic Outfits | Coalition Advanced | Coalition Basics | Common Outfits | Hai Advanced | Hai Basics | Hai Intermediate | Korath Basics | Korath Exiles | Remnant | Remnant Core | Unfettered Basics | Wanderer Advanced | Wanderer Basics
Impala Plasma Steering | Delta V | Delta V Advanced |   |   |   |   |   |   |   |   |   |   |   |   |  
Impala Plasma Thruster | Delta V | Delta V Advanced |   |   |   |   |   |   |   |   |   |   |   |   |  
Interference Plating |   | Pirate Outfits |   |   |   |   |   |   |   |   |   |   |   |   |  
Javelin | Delta V | Ammo South | Delta V Advanced | Delta V Basics | Syndicate Advanced | Syndicate Basics |   |   |   |   |   |   |   |   |  
Javelin Mini Pod | Delta V | Delta V Advanced | Delta V Basics | Syndicate Advanced |   |   |   |   |   |   |   |   |   |   |  
Javelin Pod | Delta V | Delta V Advanced | Delta V Basics | Syndicate Advanced |   |   |   |   |   |   |   |   |   |   |  
Javelin Storage Crate | Delta V | Delta V Advanced | Delta V Basics | Syndicate Advanced |   |   |   |   |   |   |   |   |   |   |  
KP-6 Photovoltaic Array |   | Common Outfits |   |   |   |   |   |   |   |   |   |   |   |   |  
KP-6 Photovoltaic Panel |   | Common Outfits |   |   |   |   |   |   |   |   |   |   |   |   |  
LP036a Battery Pack |   | Basic Outfits | Common Outfits |   |   |   |   |   |   |   |   |   |   |   |  
LP072a Battery Pack |   | Basic Outfits | Common Outfits |   |   |   |   |   |   |   |   |   |   |   |  
LP144a Battery Pack |   | Common Outfits | Deep Sky Advanced | Deep Sky Basics | Kraz Advanced | Kraz Basics | Syndicate Advanced | Syndicate Basics |   |   |   |   |   |   |  
LP288a Battery Pack |   | Deep Sky Advanced | Kraz Advanced |   |   |   |   |   |   |   |   |   |   |   |  
LP576a Battery Pack |   | Deep Sky Advanced | Kraz Advanced |   |   |   |   |   |   |   |   |   |   |   |  
Laser Rifle |   | Common Outfits | Kraz Advanced | Pirate Outfits | Syndicate Advanced | Syndicate Basics |   |   |   |   |   |   |   |   |  
Laser Turret | Deep Sky | Deep Sky Advanced | Deep Sky Basics | Kraz Advanced |   |   |   |   |   |   |   |   |   |   |  
Liquid Helium Cooler | Kraz | Kraz Advanced |   |   |   |   |   |   |   |   |   |   |   |   |  
Liquid Nitrogen Cooler | Kraz | Kraz Advanced | Kraz Basics |   |   |   |   |   |   |   |   |   |   |   |  
Luxury Accommodations |   | Coalition Advanced | Deep Sky Advanced | Hai Advanced | Lovelace Advanced | Lovelace Basics | Syndicate Advanced |   |   |   |   |   |   |   |  
Meteor Missile |   | Ammo North | Ammo South | Basic Outfits | Basic Weapons | Common Outfits | Delta V Advanced | Delta V Basics | Syndicate Advanced | Syndicate Basics |   |   |   |   |  
Meteor Missile Box |   | Basic Weapons | Common Outfits | Delta V Advanced | Syndicate Advanced | Syndicate Basics |   |   |   |   |   |   |   |   |  
Meteor Missile Launcher |   | Basic Weapons | Common Outfits | Delta V Advanced | Syndicate Advanced | Syndicate Basics |   |   |   |   |   |   |   |   |  
Meteor Missile Pod |   | Basic Weapons | Common Outfits | Delta V Advanced | Syndicate Advanced | Syndicate Basics |   |   |   |   |   |   |   |   |  
Modified Blaster | Delta V | Delta V Advanced | Pirate Outfits |   |   |   |   |   |   |   |   |   |   |   |  
Modified Blaster Turret | Delta V | Delta V Advanced | Pirate Outfits |   |   |   |   |   |   |   |   |   |   |   |  
NT-200 Nucleovoltaic | Deep Sky | Deep Sky Advanced |   |   |   |   |   |   |   |   |   |   |   |   |  
Nerve Gas |   | Pirate Outfits |   |   |   |   |   |   |   |   |   |   |   |   |  
Nuclear Missile |   | Nuke |   |   |   |   |   |   |   |   |   |   |   |   |  
Orca Plasma Steering | Delta V | Delta V Advanced |   |   |   |   |   |   |   |   |   |   |   |   |  
Orca Plasma Thruster | Delta V | Delta V Advanced |   |   |   |   |   |   |   |   |   |   |   |   |  
Outfit Scanner |   | Common Outfits | Hai Advanced | Hai Intermediate | Kraz Advanced | Lovelace Advanced | Remnant |   |   |   |   |   |   |   |  
Outfits Expansion |   | Coalition Advanced | Coalition Basics | Common Outfits | Hai Advanced | Hai Intermediate | Korath Basics | Korath Exiles | Remnant | Wanderer Advanced | Wanderer Basics |   |   |   |  
Particle Cannon | Lovelace Labs | Lovelace Advanced |   |   |   |   |   |   |   |   |   |   |   |   |  
Plasma Cannon | Kraz | Kraz Advanced | Pirate Outfits |   |   |   |   |   |   |   |   |   |   |   |  
Proton Gun | Syndicate | Syndicate Advanced |   |   |   |   |   |   |   |   |   |   |   |   |  
Proton Turret | Syndicate | Syndicate Advanced |   |   |   |   |   |   |   |   |   |   |   |   |  
Quad Blaster Turret | Syndicate | Syndicate Advanced |   |   |   |   |   |   |   |   |   |   |   |   |  
RT-I Radiothermal | Syndicate | Syndicate Advanced | Syndicate Basics |   |   |   |   |   |   |   |   |   |   |   |  
Ramscoop |   | Deep Sky Advanced | Deep Sky Basics | Northern Explorers |   |   |   |   |   |   |   |   |   |   |  
S3 Thermionic | Kraz | Kraz Advanced | Kraz Basics |   |   |   |   |   |   |   |   |   |   |   |  
Scram Drive |   | Delta V Advanced | Lovelace Advanced | Pirate Outfits | Remnant | Syndicate Advanced |   |   |   |   |   |   |   |   |  
Security Station |   | Deep Sky Advanced | Syndicate Advanced |   |   |   |   |   |   |   |   |   |   |   |  
Sidewinder Missile | Lovelace Labs | Ammo North | Lovelace Advanced | Lovelace Basics |   |   |   |   |   |   |   |   |   |   |  
Sidewinder Missile Launcher | Lovelace Labs | Lovelace Advanced | Lovelace Basics |   |   |   |   |   |   |   |   |   |   |   |  
Sidewinder Missile Pod | Lovelace Labs | Lovelace Advanced | Lovelace Basics |   |   |   |   |   |   |   |   |   |   |   |  
Sidewinder Missile Rack | Lovelace Labs | Lovelace Advanced | Lovelace Basics |   |   |   |   |   |   |   |   |   |   |   |  
Stack Core | Delta V | Stack Core |   |   |   |   |   |   |   |   |   |   |   |   |  
Supercapacitor |   | Common Outfits | Hai Advanced | Hai Basics | Hai Intermediate | Unfettered Basics |   |   |   |   |   |   |   |   |  
Tactical Scanner | Kraz | Kraz Advanced | Kraz Basics |   |   |   |   |   |   |   |   |   |   |   |  
Torpedo |   | Ammo North | Deep Sky Advanced | Deep Sky Basics | Kraz Advanced | Kraz Basics | Lovelace Advanced | Lovelace Basics | Syndicate Advanced |   |   |   |   |   |  
Torpedo Launcher |   | Deep Sky Advanced | Kraz Advanced | Lovelace Advanced | Syndicate Advanced |   |   |   |   |   |   |   |   |   |  
Torpedo Pod |   | Deep Sky Advanced | Kraz Advanced | Lovelace Advanced | Syndicate Advanced |   |   |   |   |   |   |   |   |   |  
Torpedo Storage Rack |   | Deep Sky Advanced | Kraz Advanced | Lovelace Advanced | Syndicate Advanced |   |   |   |   |   |   |   |   |   |  
Tyrant Plasma Steering | Delta V | Delta V Advanced |   |   |   |   |   |   |   |   |   |   |   |   |  
Tyrant Plasma Thruster | Delta V | Delta V Advanced |   |   |   |   |   |   |   |   |   |   |   |   |  
Volcano Afterburner | Delta V | Common Outfits | Pirate Outfits |   |   |   |   |   |   |   |   |   |   |   |  
Water Coolant System | Syndicate | Common Outfits | Syndicate Advanced |   |   |   |   |   |   |   |   |   |   |   |  
X1100 Ion Reverse Thruster | Syndicate | Syndicate Advanced | Syndicate Basics |   |   |   |   |   |   |   |   |   |   |   |  
X1200 Ion Steering | Syndicate | Basic Outfits | Common Outfits |   |   |   |   |   |   |   |   |   |   |   |  
X1700 Ion Thruster | Syndicate | Basic Outfits | Common Outfits |   |   |   |   |   |   |   |   |   |   |   |  
X2200 Ion Steering | Syndicate | Basic Outfits | Common Outfits |   |   |   |   |   |   |   |   |   |   |   |  
X2700 Ion Thruster | Syndicate | Basic Outfits | Common Outfits |   |   |   |   |   |   |   |   |   |   |   |  
X3200 Ion Steering | Syndicate | Common Outfits | Syndicate Advanced | Syndicate Basics |   |   |   |   |   |   |   |   |   |   |  
X3700 Ion Thruster | Syndicate | Common Outfits | Syndicate Advanced | Syndicate Basics |   |   |   |   |   |   |   |   |   |   |  
X4200 Ion Steering | Syndicate | Syndicate Advanced |   |   |   |   |   |   |   |   |   |   |   |   |  
X4700 Ion Thruster | Syndicate | Syndicate Advanced |   |   |   |   |   |   |   |   |   |   |   |   |  
X5200 Ion Steering | Syndicate | Syndicate Advanced |   |   |   |   |   |   |   |   |   |   |   |   |  
X5700 Ion Thruster | Syndicate | Syndicate Advanced |   |   |   |   |   |   |   |   |   |   |   |   |  
nGVF-AA Fuel Cell |   | Basic Outfits | Common Outfits |   |   |   |   |   |   |   |   |   |   |   |  
nGVF-BB Fuel Cell |   | Basic Outfits | Common Outfits |   |   |   |   |   |   |   |   |   |   |   |  
nGVF-CC Fuel Cell |   | Common Outfits | Deep Sky Advanced | Kraz Advanced | Kraz Basics |   |   |   |   |   |   |   |   |   |  
nGVF-DD Fuel Cell |   | Deep Sky Advanced | Kraz Advanced |   |   |   |   |   |   |   |   |   |   |   |  
nGVF-EE Fuel Cell |   | Kraz Advanced |   |   |   |   |   |   |   |   |   |   |   |   |  
</summary>

Thoughts?

**Comment by Amazinite** [comment]
#5726

**Comment by MasterOfGrey** [comment]
Is there an interest in potentially defining additional organisations responsible for outfit manufacture?

**Reply by Hecter94** [reply]
It would probably be best to use the already defined and well-known manufacturers unless there's a case to be made that none of the manufacturers or corporations work for thematic purposes. 
The main thing that seems to be lacking, where I think an additional manufacturer could potentially fit is with the republic Navy, most of their outfits seem to be produced and/or designed by third parties, namely Deep Sky, Lovelace Labs (probably) and Syndicate. 
I could see them having their own outfit line that they manufacture and use for their own warships, rather than relying on third parties like that.
